gitpod-init.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+ln -s "$(which ruby)" "$(rvm gemdir)/bin/ruby"
```

Comment:
For some reason, Foreman is having issue loading up 3.14. Gitpod has their own way to linking up Ruby where it's in a different location from gem dir. Symlinking fixes this problem

Reply:
![https://media.giphy.com/media/WpaVhEcp3Qo2TjwyI1/giphy.gif](https://media.giphy.com/media/WpaVhEcp3Qo2TjwyI1/giphy.gif)

Reply:
I've tested many things. If nothing else were to change but we revert Ruby, it _will_ work. To me this means gitpod is doing something under the hood and they didn't do it for ruby 3.1.4

Reply:
Any chance we could switch to using their [`workspace-full`](https://hub.docker.com/r/gitpod/workspace-full) image as a base, instead of the `postgres` image? (It's [their recommended image for ruby `3.1`](https://www.gitpod.io/docs/introduction/languages/ruby), for what it's worth.)

Reply:
@jaw6 yeah let me give that a try